sr_description/arm/config/arm_controller.yaml

```diff
@@ -34,23 +34,8 @@ sa_er_position_controller:
     d: 5.0
     i_clamp: 5.0
 
-r_arm_cartesian_pose_controller:
```

Comment:
doesn't exist in ros-control. Recommended way to do this seems to use moveit compute cartesian path from a program (which uses the trajectory controller to replay the movement).